core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala

```diff
@@ -164,18 +164,24 @@ object SnappyEmbeddedTableStatsProviderService extends TableStatsProviderService
 
   override def getStatsFromAllServers(sc: Option[SparkContext] = None): (Seq[SnappyRegionStats],
       Seq[SnappyIndexStats], Seq[SnappyExternalTableStats]) = {
+    val currTableSizeInfo = tableSizeInfo.values.toSeq
```

Comment:
No need for a separate variable currTableSizeInfo.

Reply:
Removed unnecessary variable currTableSizeInfo.